ansible/files/configuration/configuration.json

```diff
@@ -43,6 +44,11 @@
       "log4j.appender.FILE.layout": "org.apache.log4j.PatternLayout",
       "log4j.appender.FILE.layout.ConversionPattern": "%d{yy/MM/dd HH:mm:ss} %p %c{1}: %m%n"
     }
+  }, {
+    "Classification": "spark-metrics",
+    "Properties": {
+      "*.sink.statsd.class": "org.apache.spark.metrics.sink.StatsdSink"
+    }
```

Comment:
This configuration means that every Spark cluster we launch will now be running StatsdSink, but it should fail silently if no statsd agent is running on the host.